packages/mdc-chips/_variables.scss

```diff
@@ -28,6 +28,8 @@ $mdc-chip-trailing-icon-hover-opacity: .62;
 $mdc-chip-trailing-icon-focus-opacity: .87;
 $mdc-chip-leading-icon-size: 20px;
 $mdc-chip-trailing-icon-size: 18px;
+$mdc-chip-leading-icon-speedup: -50ms;
```

Comment:
Change this name to `$mdc-chip-leading-icon-delay` and move the comment about why it's negative to this file.

Reply:
Done!